test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usda

```diff
@@ -0,0 +1,312 @@
+        token outputs:surface.connect = </Looks/r_normals_reversed_x_0_bias_z/PreviewSurface.outputs:surface>
+
+        def Shader "PreviewSurface"
+        {
+            uniform token info:id = "UsdPreviewSurface"
+            color3f inputs:diffuseColor = (0.486275, 0.486275, 0.486275)
+            float inputs:metallic = 0
+            normal3f inputs:normal.connect = </Looks/r_normals_reversed_x_0_bias_z/normal_texture.outputs:rgb>
+            float inputs:opacity = 1
+            float inputs:roughness = 1
+            int inputs:useSpecularWorkflow = 0
+            token outputs:out
+            token outputs:surface
+        }
+
+        def Shader "uv_reader"
+        {
+            uniform token info:id = "UsdPrimvarReader_float2"
+            float2 inputs:fallback = (0, 0)
+            token inputs:varname = "st"
+            float2 outputs:result
+        }
+
+        def Shader "normal_texture"
+        {
+            uniform token info:id = "UsdUVTexture"
+            float4 inputs:bias = (1, -1, 0, -1)
+            asset inputs:file = @./r_normal_map_reversed_x_0_bias_z.png@
+            float4 inputs:scale = (-2, 2, 1, 2)
+            token inputs:sourceColorSpace = "raw" (
+                allowedTokens = ["auto", "raw", "sRGB"]
+            )
+            float2 inputs:st.connect = </Looks/r_normals_reversed_x_0_bias_z/uv_reader.outputs:result>
+            token inputs:wrapS = "repeat"
+            token inputs:wrapT = "repeat"
+            float3 outputs:rgb
+        }
+    }
+
+    def Material "r_normals_reversed_y"
+    {
+        token outputs:displacement.connect = </Looks/r_normals_reversed_y/PreviewSurface.outputs:displacement>
+        token outputs:surface.connect = </Looks/r_normals_reversed_y/PreviewSurface.outputs:surface>
+
+        def Shader "PreviewSurface"
+        {
+            uniform token info:id = "UsdPreviewSurface"
+            color3f inputs:diffuseColor = (0.486275, 0.486275, 0.486275)
+            float inputs:metallic = 0
+            normal3f inputs:normal.connect = </Looks/r_normals_reversed_y/normal_texture.outputs:rgb>
+            float inputs:opacity = 1
+            float inputs:roughness = 1
+            int inputs:useSpecularWorkflow = 0
+            token outputs:out
+            token outputs:surface
+        }
+
+        def Shader "uv_reader"
+        {
+            uniform token info:id = "UsdPrimvarReader_float2"
+            float2 inputs:fallback = (0, 0)
+            token inputs:varname = "st"
+            float2 outputs:result
+        }
+
+        def Shader "normal_texture"
+        {
+            uniform token info:id = "UsdUVTexture"
+            float4 inputs:bias = (-1, 1, -1, -1)
+            asset inputs:file = @./r_normal_map_reversed_y.png@
+            float4 inputs:scale = (2, -2, 2, 2)
+            token inputs:sourceColorSpace = "raw" (
+                allowedTokens = ["auto", "raw", "sRGB"]
```

Comment:
Huh, interesting, and are these a problem in some way? Omniverse spits these out by default, so I'd like to dig on this topic a bit more, in case there's something we should be avoiding on a grander (much grander) scale.

Reply:
Anyway, looks like I can remove them, so, done.